piccolo_admin/example.py

```diff
@@ -399,6 +403,14 @@ def validate_email(cls, v):
         return v
 
 
+class DownloadMoviesModel(BaseModel):
+    director_name: str
+
+
+class DownloadScheduleModel(BaseModel):
+    date: datetime.date
```

Comment:
The date field is displayed as a regular string field. I think we should add `v-bind:format="property.format"` to `NewForm` [InputField](https://github.com/piccolo-orm/piccolo_admin/blob/master/admin_ui/src/components/NewForm.vue#L8-L17). After that, the datetime picker field is displayed.

Reply:
Done